OpenRA.Mods.Common/Activities/Air/FlyCircle.cs

```diff
@@ -18,6 +18,13 @@ public class FlyCircle : Activity
 	{
 		readonly Aircraft plane;
 		readonly WDist cruiseAltitude;
+		int remainingTicks = -1;
+
+		public FlyCircle(Actor self, int ticks)
```

Comment:
Implement this as a default parameter on the normal ctor instead: `public FlyCircle(Actor self int ticks = -1)`